castervoice/apps/explorer.py

```diff
@@ -1,15 +1,42 @@
+# import pyperclip
 from dragonfly import (Grammar, MappingRule, Dictation, IntegerRef,
-                       Repeat, Pause)
+                       Repeat, Pause, Function, Choice)
 
-from castervoice.lib import control
+from castervoice.lib import control, context, utilities, settings
+from castervoice.lib.context import AppContext, paste_string_without_altering_clipboard
 from castervoice.lib import settings
```

Comment:
remove paste_string... and associated code commented out below. Always better to have something slow that works.